agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -693,6 +693,17 @@ bool ap_manager_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_
 
         std::string sta_mac = network_utils::mac_to_string(request->mac());
         std::string bssid   = network_utils::mac_to_string(request->bssid());
+
+        const auto &vap_unordered_map = ap_wlan_hal->get_radio_info().available_vaps;
+        auto it = std::find_if(vap_unordered_map.begin(), vap_unordered_map.end(),
+                               [&](const std::pair<int, bwl::VAPElement> &element) {
+                                   return element.second.mac == bssid;
+                               });
+
+        if (it == vap_unordered_map.end()) {
```

Comment:
I disagree, this should be a part of  to the "unified agent" task  that @morantr is working on right now.

Reply:
If its a task to the unified agent - then it means that until the unified agent is ready this should be in the backhaul manager, so I'm not sure why you disagree.
If you need to handle error cases this is a bug - for example in the sta measurements flow, we need to reply with an error TLV if the STA is not connected to any of the BSSes in the agent, so this is why it has to be in the backhaul manager.
Since this is part of EasyMesh steering - its the same. Anyway, it can wait to the unified agent.